frontend/src/components/Post.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import axios from "axios";
-import {Link} from "react-router-dom";
-import CommentList from "./Comment/CommentList";
-import PostComment from "./Comment/PostComment";
+import { Link } from "react-router-dom";
+import { PostComment } from "./Comment/PostComment";
 import "./Post.scss";
 import '../config';
+import CommentList from "./Comment/CommentList";
 
 class Post extends React.Component {
```

Comment:
no functional component here?

Reply:
Yes we should convert Post into a functional but my ticket was refining the comment components

Reply:
youre right i just saw the issue. should probably put that in PR title